ota-plus-web/app/reactapp/src/components/device/OverviewPanel.jsx

```diff
@@ -50,15 +48,15 @@ class OverviewPanel extends Component {
     return (
       <div className="overview-panel">
         <Tabs type="card" onChange={setOverviewPanelActiveTabId} activeKey={activeTabId}>
-          <TabPane tab={t('devices.history')} id="installation-history" key={OVERVIEW_PANEL_TAB_ID_0}>
+          <Tabs.TabPane tab={t('devices.history')} id="installation-history" key={OVERVIEW_PANEL_TAB_ID_0}>
             <HistoryMtuList device={device} />
-          </TabPane>
-          <TabPane tab={t('devices.installation_pending')} id="queued-packages" key={OVERVIEW_PANEL_TAB_ID_1}>
+          </Tabs.TabPane>
+          <Tabs.TabPane tab={t('devices.installation_pending')} id="queued-packages" key={OVERVIEW_PANEL_TAB_ID_1}>
             <QueueMtuList cancelMtuUpdate={cancelMtuUpdate} />
-          </TabPane>
-          <TabPane tab={t('devices.consent_pending')} id="approval-pending-campaigns" key={OVERVIEW_PANEL_TAB_ID_2}>
+          </Tabs.TabPane>
+          <Tabs.TabPane tab={t('devices.consent_pending')} id="approval-pending-campaigns" key={OVERVIEW_PANEL_TAB_ID_2}>
```

Comment:
Line 57 exceeds the maximum line length of 120  max-len

Reply:
done